monitoring/db_update_sqlite.py

```diff
@@ -305,53 +304,82 @@ def refresh_BenchmarksBySubmitHost_from_view(view_name):
     try:
         if view_name == 'VSummaries':
             sql_query = f"""
-            SELECT DISTINCT v.Site, v.SubmitHost, v.ServiceLevelType, v.ServiceLevel, v.UpdateTime AS LatestPublish
-            FROM {view_name} AS v
-            JOIN (
-                SELECT Site, SubmitHost, MAX(UpdateTime) AS LatestPublish
-                FROM {view_name}
-                WHERE UpdateTime > DATE_SUB(NOW(), INTERVAL 3 MONTH)
-                GROUP BY Site, SubmitHost, ServiceLevelType, ServiceLevel
-            ) AS latest
-            ON v.Site = latest.Site
-               AND v.SubmitHost = latest.SubmitHost
-               AND v.UpdateTime = latest.LatestPublish
-            WHERE v.UpdateTime > DATE_SUB(NOW(), INTERVAL 3 MONTH);
-        """
+                WITH latest AS (
+                    SELECT
+                        Site,
+                        SubmitHost,
+                        ServiceLevelType,
+                        ServiceLevel,
+                        UpdateTime,
+                        ROW_NUMBER() OVER (
+                            PARTITION BY Site, SubmitHost
+                            ORDER BY Year DESC, Month DESC, UpdateTime DESC
```

Comment:
```suggestion
                            ORDER BY UpdateTime DESC
```
why is the `Year DESC` and `Month DESC` needed? Will it not just sort out into the same order?

Reply:
We want to make sure we're looking at the data (particularly the benchmark info) most recently published for the most recent month, not just the most recently published for any month.